### PR TITLE
fix wrong run-and-move-down behaviour (#16)

### LINKED ISSUE #16: Incorrect cursor moving
@mbroedl Using `run-and-move-down` moves the cursor incorrectly. This doesn't happen when `hydrogen-python` is deactivated.

hydrogen-python-incorrect-moving

The text is the following:
```py
import re
import regex

# NOTE: Using regex for (?r) flag

from .code_manager import CodeManager
```

I'm using version `0.0.7` and my "code to expand" in the settings is:
```
else, elif, \}, \], \), except, plt\.(?!plot), fnote
```

But note that `import regex` isn't run in the gif; it's just that the cursor is moved too far.

### PR DESCRIPTION
**Pull request by mbroedl**
Fixes #16.
Changed an includes-statement to an actual comparison.

(What an odyssey, thanks @kylebarron for your help!)

### CODE REVIEW

**Comment by kylebarron** [comment]
Git can definitely be confusing at first! Getting in the practice of using different branches can definitely help. For example, if you had put stuff before on a `dev` branch, you could've started a new `run-and-move-down-dev` branch now and it might've been easier. I still get tripped up on that too!

**Comment by mbroedl** [comment]
Good shout, I will try that next time!
I'm probably just too busy with my own stuff, that I never considered the need for using branches, let alone doing PRs a lot. :/

**Comment by kylebarron** [comment]
I never used to use branches but they can really help! Now whenever I'm trying to implement almost any feature, no matter how small, I make a new branch for it

**Comment by mbroedl** [comment]
My current main purpose for using git is writing my dissertation. I wish it had distinguishable features. :D

File: lib/main.js [hunk 1]
```diff
@@ -80,7 +80,7 @@ function expandCode(editor, code) {
   // find start of executed code
   let startBufferRow = selection.start.row;
   if (!isRunWithoutMove) {
-    while (!editor.lineTextForBufferRow(startBufferRow).includes(lastCodeLineText)) {
+    while (editor.lineTextForBufferRow(startBufferRow).trim() != lastCodeLineText.trim())) {
```
[the remark this answers is not in the document]
Comment (mbroedl) [reply lib/main.js]: Ah, yes, thank you! In fact, I was wondering myself last night why the plugin didn't work.

**Comment by mbroedl** [comment]
This should be good now @nikitakit , sorry for this issue.

**Comment by nikitakit** [comment]
Thanks! I'll try to get a new release out soon.

**Pull request merged**